tests/devices/qutrit_mixed/test_qutrit_mixed_apply_operation.py

```diff
@@ -320,3 +338,228 @@ def test_broadcasted_state(self, ml_framework, wire, two_qutrit_batched_state):
 
         assert qml.math.get_interface(res) == ml_framework
         assert qml.math.allclose(res, expected)
+
+
+@pytest.mark.parametrize("subspace", [(0, 1), (0, 2), (1, 2)])
+class TestTRXCalcGrad:
+    """Tests the application and differentiation of a TRX gate in the different interfaces."""
+
+    phi = 0.325
+
+    @staticmethod
+    def compare_expected_result(phi, state, probs, subspace, g):
+        trx = qml.TRX.compute_matrix(phi, subspace)
+        trx_adj = qml.TRX.compute_matrix(-phi, subspace)
+        state = math.reshape(state, (9, 9))
+
+        expected_probs = math.diagonal(
+            np.kron(trx, np.eye(3)) @ state @ np.kron(trx_adj, np.eye(3))
+        )
+        assert qml.math.allclose(probs, expected_probs)
+
+        if subspace[0] == 0:
+            gell_mann_index = 1 if subspace[1] == 1 else 4
+        else:
+            gell_mann_index = 6
+        gell_mann_matrix = qml.GellMann.compute_matrix(gell_mann_index)
+        trx_derivative = -0.5j * gell_mann_matrix @ trx
+        trx_adj_derivative = 0.5j * gell_mann_matrix @ trx_adj
+
+        expected_derivative_state = (
```

Comment:
Nice - this result was verified by hand?

Reply:
Yes it should just be the same as the qubit version of exp(-i/2theta)